lv_objx/lv_ddlist.c

```diff
@@ -525,6 +525,8 @@ static bool lv_ddlist_design(lv_obj_t * ddlist, const lv_area_t * mask, lv_desig
 		/*Add a down symbol in ddlist*/
 		else
 		{
+			lv_ddlist_ext_t * ext = lv_obj_get_ext_attr(ddlist);	/*fix a bomb here*/
```

Comment:
Why was it necessary to duplicate code between this line and line 494?

https://github.com/littlevgl/lvgl/blob/6742d626e700f3b6ad259137af5abf70f876cd7b/lv_objx/lv_ddlist.c#L494

Reply:
I think it is a compile error due to eclipse, because i chased that error for 4 hours.
It won't work ocassionaly and make me confused for no reason, a reboot fixed that.